usage-guides/accounts-and-balances.md

```diff
@@ -4,34 +4,30 @@ To manage Blockchain Accounts, you need to visit the `Accounts` section from the
 
 ![Blockchain accounts page](/images/accounts.png)
 
-## Adding and Removing Blockchain Accounts
+## Blockchain Accounts
 
 rotki allows to track balances of blockchain accounts.
 
 For now, the following chains are supported in rotki (and the list will be growing as we add more chains in the future):
 
-- **EVM Chains**:
-
-  - Ethereum (eth)
-  - Optimism (optimism)
-  - Polygon PoS (polygon_pos)
-  - Arbitrum One (arbitrum_one)
-  - Base (base)
-  - Gnosis (gnosis)
-  - Scroll (scroll)
-  - Binance Smart Chain (binance_sc)
-  - Avalanche (avax)
-  - zkSync Lite (zksync_lite)
-
-- **Bitcoin Chains**:
-
-  - Bitcoin (btc)
-  - Bitcoin Cash (bch)
-
-- **Substrate Chains**:
-
-  - Polkadot (dot)
-  - Kusama (ksm)
+| Chain Type           | Blockchain          | Chain ID     |
+| -------------------- | ------------------- | ------------ |
+| **EVM Chains**       | Ethereum            | eth          |
+|                      | Optimism            | optimism     |
+|                      | Polygon PoS         | polygon_pos  |
+|                      | Arbitrum One        | arbitrum_one |
+|                      | Base                | base         |
+|                      | Gnosis              | gnosis       |
+|                      | Scroll              | scroll       |
```

Comment:
what is this chain ID? generally by chain ID in the evm context people understand the unique number, 1 for mainnet, 100 from gnosis and so on

Reply:
It is chain Id for importing purposes, if user want to export csv for accounts, they can use this ID. I put more information there so it's not confusing